src/components/TokenSelect/index.tsx

```diff
@@ -98,7 +102,12 @@ export function TokenSelectDialogContent({ value, onSelect, onClose }: TokenSele
     )
   }
   return (
-    <TokenSelectContainer>
+    <TokenSelectContainer
+      style={{
+        minWidth: isPageCentered ? Math.min(400, width) : 'auto',
+        minHeight: isPageCentered ? 'unset' : '100%',
+      }}
```

Comment:
can you do this with `$isPageCentered` as a transient styled-component prop and something like `min(400px, 100vw)`?

Reply:
yep